docs/source/deployment/deploy_sre.md

```diff
@@ -0,0 +1,32 @@
+(deploy_sre)=
+
+# Deploy a Secure Research Environment (SRE)
+
+## Configuration
+
+Each project will have its own dedicated Secure Research Environment (SRE).
+
+Make sure that your config file contains one or more SRE sections.
+
+:::{tip}
+You can check this by running: `dsh config show` and looking at the `sres` section.
+:::
+
+## Requirements
+
+:::{important}
+As private endpoints for flexible PostgreSQL are still in preview, the following command is currently needed:
```

Comment:
```suggestion
As private endpoints for flexible PostgreSQL are still in preview, the following command is currently needed if you are intending to deploy a PostgreSQL database:
```

Reply:
As we also use flexible PostgreSQL to back e.g. Gitea and HedgeDoc I don't think there's any way around this.